Assets/MixedRealityToolkit.SDK/StandardAssets/Materials/MRTK_ArticulatedHandMeshPulse.mat

```diff
@@ -0,0 +1,59 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
```

Comment:
Should this go into experimental, since pulse is experimental feature?

Reply:
Most likely. Otherwise, it could cause some confusion.,

Reply:
Moved materials to the Experimental folder.